fiftyone/server/view.py

```
@@ -869,6 +875,31 @@ def _apply_none(expr, f, none):
     return expr
 
 
+def _match_temporal_tags(view: foc.SampleCollection, temporal_tags):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_match_temporal_tags`

(ANN202)